src/test/regress/sql/window_functions.sql

```diff
@@ -347,6 +409,23 @@ ORDER BY
 	3 DESC, 2 DESC, 1 DESC;
 $Q$);
 
+SELECT
+	value_2,
+	AVG(avg(value_1)) OVER (PARTITION BY value_2, max(value_2), MIN(value_2)),
+	AVG(avg(value_2)) OVER (PARTITION BY value_2, min(value_2), AVG(value_1))
```

Comment:
Maybe add some tests which partition by distribution column on one window function, while partitioning by non-distribution column in another window function.